CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## v1.4.3
 
+### Bug fixes
+- `Dict`: accessing an inexistent key now raises a `KeyError` (instead of an `AttributeError`) [[#4616]](https://github.com/aiidateam/aiida-core/pull/4616)
+
+## v1.4.3
```

Comment:
The version above still says `1.4.3` and should be `1.4.4`

Reply:
I would also add the following
```
### Developers
- CI: manually install `numpy` to prevent incompatible releases [[#4615]](https://github.com/aiidateam/aiida-core/pull/4615)

Reply:
@ramirezfranciscof please incorporate the additional section to the changelog mentioned above